Test enabling full testing on 3.11 for linux

albanD opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #94056
* #94312

Testing what happens if we run everything right now.
Will remove the broken stuff to get a a mergeable version next.

albanD committed:
Test enabling full testing on 3.11 for linux

[ghstack-poisoned]

malfet commented on .github/workflows/pull.yml:
```diff
@@ -146,7 +146,13 @@ jobs:
+          { config: "default", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "crossref", shard: 1, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "crossref", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "dynamo", shard: 1, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "dynamo", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
```
This would fail, wouldn't it?

albanD replied:
Yes, I want to see how this looks right now, won't merge yet.

albanD replied:
btw all dynamo tests are already skipped today. So this should work as is. It will make it easier for William to enable these tests if the shards are already there.

malfet reviewed (approved):
If you can get it work, it would be great!

Also, please delete `smoke` if from `test.sh`

albanD committed:
Update on "[WIP] Test enabling full testing on 3.11 for linux"


Testing what happens if we run everything right now.
Will remove the broken stuff to get a a mergeable version next.


[ghstack-poisoned]

albanD committed:
Update on "[WIP] Test enabling full testing on 3.11 for linux"


Testing what happens if we run everything right now.
Will remove the broken stuff to get a a mergeable version next.


[ghstack-poisoned]

albanD committed:
Update on "[WIP] Test enabling full testing on 3.11 for linux"


Testing what happens if we run everything right now.
Will remove the broken stuff to get a a mergeable version next.


[ghstack-poisoned]

albanD committed:
Update on "[WIP] Test enabling full testing on 3.11 for linux"


Testing what happens if we run everything right now.
Will remove the broken stuff to get a a mergeable version next.


[ghstack-poisoned]

albanD committed:
Update on "[WIP] Test enabling full testing on 3.11 for linux"


Testing what happens if we run everything right now.
Will remove the broken stuff to get a a mergeable version next.


[ghstack-poisoned]

albanD committed:
Update on "[WIP] Test enabling full testing on 3.11 for linux"


Testing what happens if we run everything right now.
Will remove the broken stuff to get a a mergeable version next.


[ghstack-poisoned]

albanD committed:
Update on "[WIP] Test enabling full testing on 3.11 for linux"


Testing what happens if we run everything right now.
Will remove the broken stuff to get a a mergeable version next.


[ghstack-poisoned]

albanD commented:
@pytorchbot merge
